components/compliance-service/api/tests/04_stats_spec.rb

```diff
@@ -369,9 +369,11 @@ def stats
           {
             "name" => "linux-baseline",
             "id" => "b53ca05fbfe17a36363a40f3ad5bd70aa20057eaf15a9a9a8124a84d4ef08015",
-            "failures" => 22,
-            "criticals" => 22,
-            "passed" => 23
+            "failures" => 21,
+            "criticals" => 21,
+            "passed" => 22,
+            # COMMENT THIS OUT WHEN WAIVERS API SUPPORT IS ADDED FOR THIS API CALL
```

Comment:
nice! that'll make it easier later!

Reply:
Thank you. Wanted in a way to highlight why those expected values had to be changed.